### PR TITLE
Add a doc for cartography

### PR DESCRIPTION
**Pull request by gpetit**
Add a doc for cartography (as asked in #18)

Also add :
- author references in .rst files.
- some small improvement about subtitle (rst compilation warnings).

### CODE REVIEW

File: docs/users/attributes.rst [hunk 1]
```diff
@@ -33,7 +34,7 @@ A new tab appears in the UI with all the attributes.
 
 
 Operation on lines
```
Comment (ebocher) [inline docs/users/attributes.rst]: rows ?

File: docs/users/attributes.rst [hunk 2]
```diff
@@ -33,7 +34,7 @@ A new tab appears in the UI with all the attributes.
...
----------------
+-------------------------
 
 With a right-click on a line, the user can access to several options:
```
Comment (ebocher) [inline docs/users/attributes.rst]: row

File: docs/users/attributes.rst [hunk 3]
```diff
@@ -68,7 +69,7 @@ With a right-click on a line, the user can access to several options:
 
 
 Operation on a field
----------------
+----------------------------
 
 With a right-click on a field, the user can access to several options:
```
Comment (ebocher) [inline docs/users/attributes.rst]: column

File: docs/users/cartography.rst [hunk 4]
```diff
@@ -0,0 +1,294 @@
+.. Author : Gwendall PETIT (Lab-STICC - CNRS UMR 6285 / DECIDE Team)
+
+.. _cartography:
+
+Cartography 
+===========
+
+In OrbisGIS, you can manage the way you display geographic informations in the map thanks to the definition and the configuration of styles.
```
Comment (ebocher) [inline docs/users/cartography.rst]: OrbisGIS offers methods to produce maps using a cartographic language. The cartographic language is managed in a style model. The style model is based on research work that propose to extend the Symbology Encoding standard.

File: docs/users/cartography.rst [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+
+
+.. _Symbology Encoding: http://www.opengeospatial.org/standards/se
+
+.. _website: http://se.orbisgis.org/
+
+In OrbisGIS, styles can be exported (or imported)  as an .se file, encoded in xml.
+
+
+A style is organized as follow:
```
Comment (ebocher) [inline docs/users/cartography.rst]: defined with a set of elements.

File: docs/users/cartography.rst [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+
+* Style: a style is composed of one or *n* rule(s)
+	* Rule: a rule has one or *n* symbolizer(s)
+		* Symbolizer: a way to display geometries
```
Comment (ebocher) [inline docs/users/cartography.rst]: to renderer

File: docs/users/cartography.rst [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+
+A rule can be configured with:
+
+* a name,
+* a local langage (you can configure different languages in the same style),
+* a title *(one per langage)*,
+* an abstract *(one per langage)*,
+* Min and/or Max scale of visibility.
+
+A symbolizer can be:
```
Comment (ebocher) [inline docs/users/cartography.rst:39]: This is not a symbolizer. A symbolizer is a Point, a Polygon a LineSymbolizer.
Here we propose a pre-configured set of symbolizers to create maps.

File: docs/users/attributes.rst (hunk 2 above)
Comment (gpetit) [reply docs/users/attributes.rst]: ok

File: docs/users/attributes.rst (hunk 3 above)
Comment (gpetit) [reply docs/users/attributes.rst]: ok

File: docs/users/cartography.rst (hunk 4 above)
Comment (gpetit) [reply docs/users/cartography.rst]: ok

File: docs/users/cartography.rst (hunk 6 above)
Comment (gpetit) [reply docs/users/cartography.rst]: ok

File: docs/users/cartography.rst (hunk 5 above)
Comment (gpetit) [reply docs/users/cartography.rst]: ok

File: docs/users/cartography.rst (hunk 7 above)
Comment (gpetit) [reply docs/users/cartography.rst:39]: ok

File: docs/users/cartography.rst [hunk 8]
```diff
@@ -45,18 +38,24 @@ A rule can be configured with:
...
 A symbolizer can be:
 
-* a `Unique symbol`_, 
-* a `Value classification`_, 
-* a `Proportionnal symbol`_, 
-* an `Interval classification`_.
+* a point,
+* a line,
+* a polygon.
+
```
Comment (ebocher) [inline docs/users/cartography.rst]: a text

File: docs/users/cartography.rst [hunk 9]
```diff
@@ -45,18 +38,24 @@ A rule can be configured with:
...
+Here, we propose a pre-configured set of symbolizers to create maps:
```
Comment (ebocher) [inline docs/users/cartography.rst]: OrbisGIS proposes a set of pre-configured symbolizers to create maps

File: docs/users/cartography.rst (hunk 8 above)
Comment (gpetit) [reply docs/users/cartography.rst]: ok

File: docs/users/cartography.rst (hunk 9 above)
Comment (gpetit) [reply docs/users/cartography.rst]: ok

**Pull request merged**